### PR TITLE
New package: nextdns-1.39.4

### LINKED ISSUE #19905: Package Request: nextdns
### Description:

This project is a DNS53 to DNS over HTTPS (DoH) proxy with advanced capabilities to get the most out of NextDNS service. Although the most advanced features will only work with NextDNS, this program can work as a client for any DoH provider.

### Links:
Github: https://github.com/nextdns/nextdns

### PR DESCRIPTION
**Pull request by RunningDroid**
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

Closes #19905
Based on #30603, which replaced #20022

### CODE REVIEW

File: srcpkgs/nextdns/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+pkgname=nextdns
+version=1.37.11
+revision=1
+build_style=go
+go_import_path="github.com/nextdns/nextdns"
+short_desc="NextDNS CLI client (DoH Proxy)"
+maintainer="RunningDroid <runningdroid@zoho.com>"
+license="MIT"
+homepage="https://nextdns.io/"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/nextdns/template:11]:
```suggestion
distfiles="https://github.com/nextdns/nextdns/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/nextdns/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+checksum=f70cb424b0ae47456c8579d3910b486f75afde34572f28f8cf6eb1222e59b88b
+
+post_patch() {
+	vsed -i main.go -e "s/version  = \"dev\"/version  = \"${version}\"/"
```
Comment (classabbyamp) [inline srcpkgs/nextdns/template:15]: this should probably be done with go ld flags (unless that doesn't work?)

Comment (RunningDroid) [reply srcpkgs/nextdns/template]: I wasn't aware I could do that, thanks for the tip!

**Comment by RunningDroid** [comment]
The patch I've added is a hacky fix for nextdns/nextdns#746

**Pull request merged**